plugin/trino-resource-group-managers/src/main/java/io/trino/plugin/resourcegroups/db/FlywayMigration.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.resourcegroups.db;
+
+import com.google.inject.Inject;
+import io.airlift.log.Logger;
+import org.flywaydb.core.Flyway;
+import org.flywaydb.core.api.output.MigrateResult;
+
+import javax.annotation.PostConstruct;
+
+public class FlywayMigration
+{
+    private static final Logger log = Logger.get(FlywayMigration.class);
+
+    private final Flyway flyway;
+
+    @Inject
+    public FlywayMigration(DbResourceGroupConfig config)
+    {
+        flyway = Flyway.configure()
+                .dataSource(config.getConfigDbUrl(), config.getConfigDbUser(), config.getConfigDbPassword())
+                .locations("/db/migration/mysql")
+                .baselineOnMigrate(true)
```

Comment:
```suggestion
                .baselineOnMigrate(true)
                .baselineVersion("0")
```
When migrating against a non-empty schema with no schema history table, the schema is baselined with version 1, by default.  Since your migration (`V1__base_resource_groups_schema.sql`) has version 1, it would get skipped, in that scenario.